openfecwebapp/templates/partials/legal-search-results-advisory-opinion.html

```diff
@@ -0,0 +1,8 @@
+<tr>
+  <td><a title="{{ advisory_opinion.description }}" href="http://saos.fec.gov/aodocs/{{ advisory_opinion.no }}.pdf#search={{ query }}">{{ advisory_opinion.no }}</a></td>
```

Comment:
Yeah, the title is intentional for now. We don't have the real document to link to ATM and I didn't want to include the full text in the table. Using the title is just to give the user an idea of what the document text is so they have an idea why the document was a match in their search. Sort of a prototype state.

If you have other ideas, I'm all ears. Doesn't look like we'll have the document urls for this next sprint so it'll be in this hacky state for a while.